### PR TITLE
Geometric control velocity and acceleration mode working

### PR DESCRIPTION
**Pull request by aliagilbert**

### CODE REVIEW

**Review by hardikparwana — APPROVED** [review]
Same as other review. Correct Indentations and then push to master branch

File: include/dasc_robot/DASCRobots.hpp [hunk 1]
```diff
@@ -86,6 +88,8 @@ class DASCRobot : public DASC {
         void emergencyStop();
         bool takeoff();
         bool land();
+        uint64_t get_current_timestamp(); // DEV: in what units? DEV: i thinks ms
```
Comment (hardikparwana) [inline include/dasc_robot/DASCRobots.hpp:91]: yeah, this is in milli seoconds

Comment (dev10110) [reply include/dasc_robot/DASCRobots.hpp:91]: Nope. Based on the experiments it's in us. 

@chenrc98 please confirm.

**Comment by dev10110** [comment]
@dev10110 fix integrator reset.

**Comment by dev10110** [comment]
superseded by #7

**Pull request closed without merging**